package-lock.json

```diff
@@ -14,8 +14,11 @@
         "@web/test-runner-playwright": "^0.10"
       },
       "devDependencies": {
+        "chai": "^4.3.7",
         "eslint": "^8",
-        "eslint-config-brightspace": "^0.23"
+        "eslint-config-brightspace": "^0.23",
+        "mocha": "^10.2.0",
+        "sinon": "^15.1.0"
```

Comment:
Might need to regenerate this. I _think_ the `update-package-lock` would just fix it, but also wouldn't be surprised if it complained loudly that things were out of sync